examples2d/add_remove2.rs

```diff
@@ -31,7 +36,11 @@ pub fn init_world(testbed: &mut Testbed) {
             physics
                 .bodies
                 .remove(handle, &mut physics.colliders, &mut physics.joints);
-            graphics.remove_body_nodes(window, handle);
+
+            // FIXME: need a way to access graphics & window in a loop
+            // if graphics.is_some() {
+            //     graphics.unwrap().remove_body_nodes(window.unwrap(), handle);
+            // }
```

Comment:
We can do the same as mentioned above:

```suggestion
            if let (Some(graphics), Some(window)) = (&mut graphics, &mut window) {
                 graphics.remove_body_nodes(*window, handle);
            }
```